### PR TITLE
feat:login signup implemented

### PR DESCRIPTION
**Pull request by aaman123**
### What this PR does (required):
- Refactors login and signup pages according to the wireframe provided in the drive folder

### Screenshots / Videos (required):
Screenshot (1542)

### Any information needed to test this feature (required):
- Nope

### Any issues with the current functionality (optional):
- Certain elements are not yet responsive, will fix them up in upcoming commits for the same PR.

### CODE REVIEW

File: client/.prettierignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#if you like to use prettier extension, please comment out the lines below
+*.tsx
+*.ts
```
Comment (ugur-akin) [inline client/.prettierignore:3]: Is there a reason why we shouldn't use prettier (or equivalent) consistently? I think consistent use of a linter/formatter will result in a much cleaner codebase.

File: client/src/components/AuthHeader/AuthHeader.tsx [hunk 2]
```diff
@@ -16,11 +16,16 @@ const AuthHeader = ({ linkTo, asideText, btnText }: Props): JSX.Element => {
   return (
     <Box p={1} className={classes.authHeader}>
       <Typography className={classes.accAside}>{asideText}</Typography>
-      <Link to={linkTo} className={classes.link}>
-        <Button color="inherit" className={classes.accBtn} variant="contained">
+      {
+        btnText === "Create account" ?
```
Comment (ugur-akin) [inline client/src/components/AuthHeader/AuthHeader.tsx:20]: Even though it's only two states, I would still suggest refactoring auth page info such as `btnText`, `asideText` and `linkTo`  into an [enum-like object](https://www.sohamkamani.com/javascript/enums/). That way comparisons will be more robust and future changes simpler.

File: client/src/pages/SignUp/SignUpForm/useStyles.ts [hunk 3]
```diff
@@ -27,7 +27,7 @@ const useStyles = makeStyles((theme) => ({
     borderRadius: theme.shape.borderRadius,
     marginTop: 49,
     fontSize: 16,
-    backgroundColor: '#3a8dff',
+    backgroundColor: '#dc3545',
```
Comment (ugur-akin) [inline client/src/pages/SignUp/SignUpForm/useStyles.ts:30]: Consider extending the Material-UI theme/palette for these colors.

**Review by ugur-akin — COMMENTED** [review]
Just some minor suggestions, otherwise LGTM!

File: client/src/components/NavBar/NavBar.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import { AppBar, Typography } from '@material-ui/core';
+import logo from '../../Images/logo.png';
+
+const NavBar = (): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <AppBar position="static" className={classes.navBar}>
+          <img className={classes.heroImage} src={logo} />
+          <div className={classes.navText}>
```
Comment (rajivtitus) [inline client/src/components/NavBar/NavBar.tsx:12]: You can use a Material UI `<Box>` component here, it also has flexbox built into it :)

Ref: https://material-ui.com/components/box/

**Review by rajivtitus — COMMENTED** [review]
Please have a look at comments and address when possible. Thank you :)

File: client/src/components/NavBar/NavBar.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import Button from '@material-ui/core/Button';
+import useStyles from './useStyles';
+import { AppBar, Typography } from '@material-ui/core';
+import logo from '../../Images/logo.png';
+
+const NavBar = (): JSX.Element => {
```
Comment (rajivtitus) [inline client/src/components/NavBar/NavBar.tsx:6]: Also, please check with Ibsa as he has been assigned the task for creating the 'Dashboard Skeleton aka Navbar'. Just to make sure we are not duplicating components, thanks!

Comment (aaman123) [reply client/src/components/NavBar/NavBar.tsx:6]: sure Rajiv , let me confirm with him ...

**Pull request merged**